### PR TITLE
(maint): update jdbc-util version for jsonb methods

### PR DESCRIPTION
**Pull request by klawhorn**

### CODE REVIEW

**Comment by jonathannewman** [comment]
Can you also update the changelog ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [2.6.2]
+
+- Update jdbc-utils to 1.2.4, which includes methods for jsonb type PGObject transormations
```
Comment (jonathannewman) [inline CHANGELOG.md]: typo:  transormations -> transformations

**Comment by jonathannewman** [comment]
Once the typo is fixed, I think this is GTG.

File: CHANGELOG.md (hunk 1 above)
Comment (klawhorn) [reply CHANGELOG.md]: gah updating

**Comment by jonathannewman** [comment]
LGTM! Don't sweat it

**Pull request merged**